stdlib/Profile/src/Profile.jl

```diff
@@ -55,7 +55,7 @@ function init(; n::Union{Nothing,Integer} = nothing, delay::Union{Nothing,Real}
     n_cur = ccall(:jl_profile_maxlen_data, Csize_t, ())
     delay_cur = ccall(:jl_profile_delay_nsec, UInt64, ())/10^9
     if n === nothing && delay === nothing
-        return Int(n_cur), delay_cur
+        return round(Int, n_cur / Threads.nthreads()), delay_cur
```

Comment:
Should this be `round(Int, n_cur), delay_cur` on Windows, and `round(Int, n_cur / Threads.nthreads()), delay_cur` on non-Windows?

Reply:
Ah, yes, good point. I'll fix that